packages/dialtone-vue2/components/tooltip/tooltip.stories.js

```diff
@@ -95,6 +95,11 @@ export const argTypesData = {
   },
 
   // Events
+  showTooltip: {
+    table: {
+      disable: true,
+    },
+  },
```

Comment:
Whats the purpose to add this?

Reply:
It seems to be some special prop only used by percy, so we don't want to show it in the storybook list.